frequency: 50
sensor_timeout: 0.1
transform_time_offset: 0.0
two_d_mode: true
print_diagnostics: false

debug: false
debug_out_file: /path/to/debug/file.txt

map_frame: map
odom_frame: odom
base_link_frame: base_link
world_frame: odom

use_control: true
stamped_control: false
control_timeout: 0.2
control_config: [true, false, false, false, false, true]
acceleration_limits: [1.3, 0.0, 0.0, 0.0, 0.0, 3.4]
deceleration_limits: [1.3, 0.0, 0.0, 0.0, 0.0, 4.5]
acceleration_gains: [0.8, 0.0, 0.0, 0.0, 0.0, 0.9]
deceleration_gains: [1.0, 0.0, 0.0, 0.0, 0.0, 1.0]

odom0: robot_publisher/odom0
odom0_config: [true,true,false,
               false, false, true,
               true,  true,  false,
               false, false, true,
               true,  true, false]
odom0_differential: false
odom0_relative: false
odom0_queue_size: 10
odom0_nodelay: false

imu0: robot_publisher/imu0
imu0_config: [false, false, false,
              true,  true,  true,
              false, false, false,
              true,  true,  true,
              true,  true,  true]
imu0_nodelay: false

odom1_differential: false
odom1_relative: true
imu0_queue_size: 10
